addons/persistence/infinispan-persistence-addon/pom.xml

```
@@ -113,19 +113,11 @@
       <plugin>
         <groupId>org.apache.maven.plugins</groupId>
         <artifactId>maven-failsafe-plugin</artifactId>
-        <executions>
-          <execution>
-            <goals>
-              <goal>integration-test</goal>
-              <goal>verify</goal>
-            </goals>
-            <configuration>
-              <systemPropertyVariables>
-                <container.image.infinispan>${container.image.infinispan}</container.image.infinispan>
-              </systemPropertyVariables>
-            </configuration>
-          </execution>
-        </executions>
+        <configuration>
+          <systemPropertyVariables>
```

```suggestion
          <systemPropertyVariables combine.children="append">
```
This makes sure `maven.repo.local` and settings.xml configuration is passed through to quarkus-based tests.